### PR TITLE
#103 retrieve page waits forever

### PR DESCRIPTION
**Pull request by anil-sfdc**
# PR details

## What is the purpose of this pull request? (put an "X" next to an item)
- [ ] Bug fix
 Apex class changes 
 1. added new class for event call callback

## What changes did you make? (Give an overview)

Platform event changes based on copada team

### CODE REVIEW

File: force-app/main/default/classes/RetriveCompletion.cls [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/*
+ * This class is used by copado JobExecution and publish the PE copado__Event__e
+ * 
+ *  Platform Event PE copado__Event__e
+ */
+global with sharing class RetriveCompletion implements copado.JobExecutionCallback {
```
Comment (JoernBerkefeld) [inline force-app/main/default/classes/RetriveCompletion.cls:6]: hard no.
please fix the class name's spelling and while you are at it, add the MCDO_ prefix to it:
MCDO_RetrieveCompletion


did we forget this component in the previous PR?

File: force-app/main/default/classes/RunCopadoFunctionFromLWC.cls [hunk 2]
```diff
@@ -124,16 +125,15 @@ public with sharing class RunCopadoFunctionFromLWC {
...
                 copado.CreateExecution.Request request = new copado.CreateExecution.Request();
                 request.parentId = userStoryId; 
                 request.templateName = templateName; 
                 request.pipelineId = pipelineId;
-                request.sourceId = envId; 
+                request.sourceId = envId;            
                 request.runAfterInstantiation = true; 
-
+               // RetriveCompletion is Class used as copada Callback
+                request.callback = 'RetriveCompletion';
```
Comment (JoernBerkefeld) [inline force-app/main/default/classes/RunCopadoFunctionFromLWC.cls:135]: after changing the class name, fix it here as well

**Review by JoernBerkefeld — CHANGES_REQUESTED** [review]

File: force-app/main/default/classes/RetriveCompletion.cls (hunk 1 above)
Comment (anil-sfdc) [reply force-app/main/default/classes/RetriveCompletion.cls:6]: Yes , part of previous one

**Review by JoernBerkefeld — CHANGES_REQUESTED** [review]

File: force-app/main/default/classes/RetriveCompletion.cls (hunk 1 above)
Comment (JoernBerkefeld) [reply force-app/main/default/classes/RetriveCompletion.cls:6]: Also, did you Update the callback to this in the salesforce interface? I believe this was done on the function "Marketing Cloud Retrieve"

**Review by anil-sfdc — COMMENTED** [review]
wrong files added  ->TestCopadoTrailHelper
I removed it

**Review by JoernBerkefeld — APPROVED** [review]

**Pull request merged**